engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/content-hosts-bulk-packages-modal.html

```diff
@@ -38,8 +37,6 @@ <h4 data-block="modal-header" translate>Update Packages</h4>
         <span class="caret"></span>
       </button>
       <ul uib-dropdown-menu role="menu" aria-labelledby="install-use-remote-execution">
-        <li ng-show="katelloAgentPresent" role="presentation"><a ng-click="performViaKatelloAgent('update all', content)" role="menuitem" tabindex="-1" href="#" translate>via Katello Agent</a></li>
-        <li role="presentation"><a ng-click="performViaRemoteExecution('update all', false)" role="menuitem" tabindex="-1" href="#" translate>via remote execution</a></li>
```

Comment:
Why remove the immediate remote execution options?

Reply:
I was removing it out of the carat dropdown so that only customize would be in the dropdown and clicking the button directly would still invoke REX. I can add it back if you want, I just didn't see the point of it being in the drop down as well as working when you click the install/update/remove button

Reply:
On the new host details page we have both options in the dropdown, so I'd say let's keep it consistent.